File: 2017/src/test/java/aoc2017/Day13Test.java
package aoc2017;

import aoc2017.Day13;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day13Test {
    @Test
    public void testPart1() {
        assertEquals(24, Day13.part1("0: 3\n" +
                "1: 2\n" +
                "4: 4\n" +
                "6: 4"));
    }

    @Test
    public void testPart2() {
        assertEquals(10, Day13.part2("0: 3\n" +
                "1: 2\n" +
                "4: 4\n" +
                "6: 4"));
    }
}

File: aoc2017-2018/src/test/java/aoc2018/Day13Test.kt
package aoc2018

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class Day13Test {

    @Test
    fun part1() {
        val input = " /n ".split("\n")
        assertThat(Day13.part1(input)).isEqualTo(1)
    }

    @Test
    fun part2() {
        val input = " /n ".split("\n")
        assertThat(Day13.part2(input)).isEqualTo(2)
    }
}
